hearth/lib/hearth/http/api_error.rb

```diff
@@ -9,7 +9,6 @@ def initialize(http_resp:, **kwargs)
         @http_status = http_resp.status
         @http_headers = http_resp.headers
         @http_body = http_resp.body
-        @request_id = http_resp.headers['x-request-id']
```

Comment:
This is considered "common" and not "standard" in http spec, so I opted to remove it here.